src/lib/bootloader/main_dialog.rb

```diff
@@ -52,7 +52,7 @@ def run
         "ws_start" => "read",
         "read"     => { abort: :abort, next: "main" },
         "main"     => { abort: :abort, next: "write" },
-        "write"    => { abort: :abort, next: :next }
+        "write"    => { abort: "main", next: :next }
```

Comment:
so if you click on abort during write it will return you to main? Is it required?

Reply:
then I suggest to use different symbol like `:cancel` and not :abort as abort is used also when user click on abort.

Reply:
Thank you @jreidinger 

The thing is how to distinguish when a false from Bootloader.Write is "translated" to `:cancel` and when to `:abort`.

I will think about it. Suggestions are welcome :)

Reply:
well, not so easy. so lets keep it as it is. Maybe just keep here abort and see if someone complains.

Reply:
@jreidinger  just to clarify, you mean to keep it as it was

```ruby
"write"    => { abort: :abort, next: :next }
```

right?

Reply:
yes